packages/account/src/Styles/account.scss

```diff
@@ -628,9 +627,14 @@ $MIN_HEIGHT_FLOATING: calc(
         &__list {
             &-container {
                 margin-top: 1.6rem;
-                max-width: 50.6rem;
+                width:100%;
```

Comment:
Can you please run prettify as it seems like it's not formatted properly

Reply:
Strange. I have to manually run it